CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### New features
 
-* **Only print `safe_name` when referencing `Target`s** ([#1271](https://github.com/puppetlabs/bolt/pull/1271))
+* **Only print `safe_name` when referencing `Target`s** ([#1243](https://github.com/puppetlabs/bolt/issues/1243))
```

Comment:
Recommend rewording to avoid confusion around the scope of the word "only":

```suggestion
* **When referring to `Target`s in log or output, use the `safe_name`** ([#1243](https://github.com/puppetlabs/bolt/issues/1243))
```